src/components/Login/index.jsx

```diff
@@ -26,34 +27,35 @@ export default class Login extends React.Component {
 
     if (userName === "ziv" && password === "papa") {
       this.setState(prevState => ({
-        isAuthenticated: !prevState.isAuthenticated,
+        isAuthenticated: !prevState.isAuthenticated
       }));
     } else {
       this.setState(prevState => ({
-        isShowingError: !prevState.isShowingError,
+        isShowingError: !prevState.isShowingError
       }));
     }
   }
 
   render() {
     const { isAuthenticated, isShowingError } = this.state;
+    const redirectLink = "/bounce_rules";
 
     return isAuthenticated ? (
-      <Redirect to="/bounce_rules" />
+      <Redirect to={redirectLink} />
     ) : (
       <div
         className="row"
         style={{
           display: "flex",
           justifyContent: "center",
           alignItems: "center",
-          height: "100vh",
+          height: "100vh"
```

Comment:
once we have more of the styles figured out, let's consolidate some shared ones into a sass file that we'll import in and use those classes for layout but for now it's okay